src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/EditingPreferencesPane.java

```diff
@@ -64,7 +64,14 @@ public EditingPreferencesPane(UIPrefs prefs,
             "Continue comment when inserting new line",
             prefs_.continueCommentsOnNewline(),
             "When enabled, pressing enter will continue comments on new lines. Press Shift + Enter to exit a comment."));
-      editingPanel.add(checkboxPref("Enable vim editing mode", prefs_.useVimMode()));
+      
+      vim_ = checkboxPref("Enable Vim editing mode", prefs_.useVimMode());
```

Comment:
Indeed they are -- I do have the appropriate wiring to ensure only one can be checked but I agree it makes much more sense to just use a combo box.